TwoWayBinding/TwoWayBinding/EditProfileViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  EditProfileViewController.swift
+//  TwoWayBinding
+//
+//  Created by Benjamin Wu on 11/28/16.
+//  Copyright © 2016 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import Intrepid
+
+class EditProfileViewController: UIViewController, UIPickerViewDataSource, UIPickerViewDelegate {
+
+    @IBOutlet private weak var usernameTextField: UITextField!
+    @IBOutlet private weak var emailTextField: UITextField!
+    @IBOutlet private weak var firstNameTextField: UITextField!
+    @IBOutlet private weak var lastNameTextField: UITextField!
+    @IBOutlet private weak var dateOfBirthTextField: UITextField!
+    @IBOutlet private weak var genderTextField: UITextField!
+    @IBOutlet private weak var logButton: UIButton!
+
+    let profile = Profile()
+    let viewModel: EditProfileViewModel
+
+    let datePicker = UIDatePicker()
+    let genderPicker = UIPickerView()
+
+    private let disposeBag = DisposeBag()
+    private let dismissTapGesture = UITapGestureRecognizer()
+
+    init(profile: Profile) {
+        viewModel = EditProfileViewModel(profile: profile)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    override convenience init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        self.init(profile: Profile())
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        dismissTapGesture.addTarget(self, action: #selector(EditProfileViewController.endEditing))
+        view.addGestureRecognizer(dismissTapGesture)
+
+        usernameTextField.rx.text <-> viewModel.username >>> disposeBag
```

Comment:
I think it depends on the class.  In this case I felt that there were enough custom set up operations for the other two fields, that it would be better to organize them on their own